examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml

```diff
@@ -0,0 +1,91 @@
+# Sentence classification with pretrained BERT models
+
+trainer:
+  gpus: 1 # the number of gpus, 0 for CPU
+  num_nodes: 1
+  max_epochs: 100
+  max_steps: null # precedence over max_epochs
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  amp_level: O0 # O1/O2 for mixed precision
+  distributed_backend: ddp
+  checkpoint_callback: False  # Provided by exp_manager
```

Comment:
Please add these to the trianer config:
  precision: 16 # Should be set to 16 for O1 and O2, default is 16 as PT ignores it when am_level is O0
  distributed_backend: ddp
  row_log_interval: 1  # Interval of logging.
  val_check_interval: 1.0  # Set to 0.25 to check 4 times per epoch, or an int for number of iterations
  resume_from_checkpoint: null # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step, LR schedulers, apex, etc.